containers/src/Kokkos_Vector.hpp

```diff
@@ -74,15 +74,19 @@ class vector : public DualView<Scalar*, LayoutLeft, Arg1Type> {
 
  public:
 #ifdef KOKKOS_ENABLE_CUDA_UVM
-  KOKKOS_INLINE_FUNCTION reference operator()(int i) const {
+  KOKKOS_FUNCTION reference operator()(int i) { return DV::h_view(i); };
+  KOKKOS_FUNCTION const_reference operator()(int i) const {
```

Comment:
While you're at it, you might as well propagate `constexpr` so that whenever we get around to fixing that up in `View` this works here as well: 

```suggestion
  KOKKOS_FUNCTION KOKKOS_CONSTEXPR_14 reference operator()(int i) { return DV::h_view(i); };
  KOKKOS_FUNCTION constexpr const_reference operator()(int i) const {
```

(The same applies throughout, but I'm not going to mark all of them.)

Reply:
`Kokkos::View::operator()` is not declared with the constexpr specifier.
I could only make it happen for `Vector::data()`.

Reply:
It should compile because `operator()` is a dependent template, so `constexpr` can be there even if in every instantiation there isn't a `constexpr` instantiation of the method. I would be very surprised if that doesn't compile.